nixos/modules/programs/wayland/waybar.nix

```diff
@@ -11,12 +16,6 @@ in
 
   config = lib.mkIf cfg.enable {
     environment.systemPackages = [ cfg.package ];
-    systemd.user.services.waybar = {
-      description = "Waybar as systemd service";
-      wantedBy = [ "graphical-session.target" ];
-      partOf = [ "graphical-session.target" ];
-      script = "${cfg.package}/bin/waybar";
-    };
   };
```

Comment:
We should also add `cfg.package` to `systemd.packages`

Reply:
Don't packages in systemPackages get included already? I need to double check.

Reply:
Added it.